rest/taskana-rest-spring/src/main/java/pro/taskana/task/rest/TaskQueryFilterParameter.java

```diff
@@ -740,6 +740,13 @@ public class TaskQueryFilterParameter implements QueryParameter<TaskQuery, Void>
    */
   @JsonProperty("owner-not-like")
   private final String[] ownerNotLike;
+
+  /**
+   * Filter by tasks that have no owner. The parameter should exactly be "owner-is-null" without
+   * being followed by "=..."
+   */
+  @JsonProperty("owner-is-null")
+  private final String[] ownerNull;
```

Comment:
This should be String instead of Array of Strings. You can only specify "ownerNull" once